CHANGELOG.md

```diff
@@ -1,5 +1,160 @@
+- Encoding for `New-ModuleManifest` on all platforms should be `UTF-8 NoBOM` (#5923)
+- Make `Set-Location` use path with wildcard characters if it exists (#5839)
+- Combine Web Cmdlets partial class files (#5612) (Thanks @markekraus!)
+- Change `Microsoft.PowerShell.Commands.SetDateCommand.SystemTime` to `struct`. (#6006) (Thanks @stknohg!)
+- Add Simplified `multipart/form-data` support to Web Cmdlets through `-Form` parameter (#5972) (Thanks @markekraus!)
+- Make a relative redirect URI absolute when `Authorization` header present (#6325) (Thanks @markekraus!)
+- Make relation-link handling in Web Cmdlets case-insensitive (#6338)
+- Make `Get-ChildItem -LiteralPath` accept `Include` or `Exclude` filter (#5462)
+- Stop `ConvertTo-Json` when `Ctrl+c` is hit (#6392)
+- Make `Resolve-Path -Relative` return useful path when `$PWD` and `-Path` is on different drive (#5740) (Thanks @kwkam!)
+- Correct the `%c`, `%l`, `%k`, `%s` and `%j` formats in `Get-Date -UFormat` (#4805) (Thanks @iSazonov!)
+- Add standard deviation implementation on `Measure-Object` (#6238) (Thanks @CloudyDino!)
+- Make `Get-ChildItem <PATH>/* -file` include `<Path>` as search directory (#5431)
+- Enable setting `PSSession` Name when using `SSHTransport` and add `Transport` property (#5954)
+- Add `Path` alias to `-FilePath` parameters and others for several commands (#5817) (Thanks @KevinMarquette!)
+- Add the parameter `-Password` to `Get-PfxCertificate` (#6113) (Thanks @maybe-hello-world!)
+- Don't add trailing spaces to last column when using `Format-Table` (#5568)
+- Fix table alignment and padding. (#6230)
+- Add `-SkipHeaderValidation` Support to `ContentType` on Web Cmdlets (#6018) (Thanks @markekraus!)
+- Add common aliases for all `write-*` commands default message parameter (#5816) (Thanks @KevinMarquette!)
+- Make `UTF-8` the default encoding for `application/json` (#6109) (Thanks @markekraus!)
+- Enable `$env:PAGER` to work correctly if arguments are used (#6144)
+- Remove the unnecessary `AddTypeCommandBase` class (#5407) (Thanks @iSazonov!)
+
+### Test
+
+- Convert Web Cmdlets test to `one-true-brace-style` formatting (#5716) (Thanks @markekraus!)
+- Add a test for `IValidateSetValuesGenerator` in a module (#5830) (Thanks @iSazonov!)
+- Fix function to test for docker OS due to change to use `linuxkit` for macOS (#5843)
+- Replace `HttpListener` link header tests with `WebListener` (#5806) (Thanks @markekraus!)
+- Replace `HttpListener` echo tests with `WebListener` (#5840) (Thanks @markekraus!)
+- Replace `HttpListener` redirect tests with `WebListener` (#5872) (Thanks @markekraus!)
+- Stop `HttpListener` from running in Web Cmdlets tests (#5921) (Thanks @markekraus!)
+- Fix `PSVersion` in `PSSessionConfiguration` tests (#5554) (Thanks @iSazonov!)
+- Update test framework and tests to support Pester version 4 (#6064)
+- Update tests in `Microsoft.PowerShell.Management` folder to use Pester 4 Syntax. (#6294)
```

Comment:
I think we should squash the Pester v4 syntax ones to just "Update tests to use Pester v4 syntax" listing out all the PR #s